CodeStarterCamp_Week2/main.swift

```diff
@@ -12,44 +12,75 @@ import Foundation
 //2. Collection 타입 중 하나를 선택해, 6개의 랜덤한 숫자를 저장합니다.
 //3. 6개의 숫자는 ‘서로 겹치지 않아야’ 합니다.
 //4. 이 함수는 6개의 랜덤한 숫자를 저장한 Collection 타입의 값을 반환합니다.
-var lottoNumbers: Set<Int> = []
+var lottoNumbers: Set<Int> = Set<Int>()
 let myLottoNumbers: Set<Int> = [1,2,3,4,5,6]
+var round : Int = 1
+var winningLottoNumbers: [String: [Int]] = [:]
+var countWinningLottoNumbers: String = ""
 
-func makeLottoNumbers(){
-    while lottoNumbers.count < 6 {
-    lottoNumbers.insert(Int.random(in: 1...45))
-}
-    print(lottoNumbers.sorted())
+func countUpLottoNumbers() {
+    round += 1
+    makeLottoNumbers()
 }
-makeLottoNumbers()
 
-//생성된 lottoNumbers와 myLottoNumbers를 비교해 공통의 숫자가 있는지 없는지를 비교
-//공통된 숫자를 뽑는 변수를 생성
-
-var matchNumber = lottoNumbers.intersection(myLottoNumbers)
-var matchStringArray: [String] = []
-
-func changeType(set: Set<Int>) -> String {
-    for index in matchNumber {
-    matchStringArray.append(String(index))
+func saveDictionary (key: String, value : [Int]) {
+    countWinningLottoNumbers = "\(round)회차"
+    winningLottoNumbers.updateValue(value, forKey: key)
 }
-    return matchStringArray.joined(separator: ", ")
-}
-
-let changedTypeLottoNumbers = changeType(set: matchNumber)
 
-func resultLotto() {
-    if matchNumber.isEmpty {
-        print("아쉽지만 겹치는 번호가 없습니다.")
+func makeLottoNumberFiveTimes() {
+    for _ in 1...5 {
+        countUpLottoNumbers()
+        saveDictionary(key: countWinningLottoNumbers, value: Array(lottoNumbers))
+        print(lottoNumbers)
+        lottoNumbers.removeAll()
+    }
 }
-    else { //배열값이 아니라 숫자가 들어가야한다.
-    print("축하합니다! 겹치는 번호는 \(changedTypeLottoNumbers) 입니다.")
+
+func checkWinningNumber(key: Int) {
+    var value: [String] = []
+    guard var unwrapValue : [Int] = winningLottoNumbers["\(key)회차"] else {return}
```

Comment:
`value`, `unwrapValue `라는 변수명은 이 코드를 처음 보는 사람은 이해하기 어려울 것입니다.
또한 [Swift API Guideline](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage)에는 모호한 단어를  피하고, 타입과 관련된 단어도 변수명으로 지양하라고 안내하고 있습니다.
> - Include all the words needed to avoid ambiguity for a person reading code where the name is used.
> - Name variables, parameters, and associated types according to their roles, rather than their type constraints.